tools/assist.py

```diff
@@ -0,0 +1,177 @@
+#!/usr/bin/python3
```

Comment:
👀 what's this?

Reply:
This is a script I've used a few times that I believe could help many people. You give it a function and then it returns similar functions in the rom, and whether or not they're decompiled. I didn't mean to include it in this PR specifically, but now that it's in, I might as well just leave it I think. I'll edit the PR description to include that though.